ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Mempool.hs

```diff
@@ -18,12 +19,14 @@
 {-# OPTIONS_GHC -Wno-orphans #-}
 
 -- | Shelley mempool integration
+--
+-- TODO nearly all of the logic in this module belongs in cardano-ledger, not
+-- ouroboros-consensus; ouroboros-consensus-cardano should just be "glue code".
```

Comment:
Seconding this; there even are potential ideas already to change which ref scripts to consider when calculating the size see https://github.com/IntersectMBO/cardano-ledger/pull/4550#issuecomment-2298275991 or https://github.com/IntersectMBO/cardano-ledger/pull/4550#issuecomment-2299748756.